fontir/src/ir.rs

```diff
@@ -42,16 +45,12 @@ impl StaticMetadata {
 pub struct Axis {
     pub name: String,
     pub tag: String,
-    pub min: OrderedFloat<f32>,
-    pub default: OrderedFloat<f32>,
-    pub max: OrderedFloat<f32>,
+    pub min: UserSpaceCoord,
+    pub default: UserSpaceCoord,
+    pub max: UserSpaceCoord,
```

Comment:
I'm a little confused about the from key reference in this context? - IR is not obliged to mirror what UFO does. I wonder if we are conflating what designspace/ufo do with what IR does?

Reply:
In looking at the python I suspect it's far far easier to become confused than it will be in rust because there are neither types nor comments on coordinates inputs or outputs are in . In rust it'll be very clear that, for example, `f(DesignCoord) -> UserCoord`.

#56 aspires to resolve this in a mutually satisfactory way by making exlicit types with to_other_type methods. Let's resolve here and try to push that through to something we can live with.